src/ad-cid.js

```diff
@@ -29,13 +29,28 @@ export function getAdCid(adElement) {
   if (!config || !config.clientIdScope) {
     return Promise.resolve();
   }
-  const cidPromise = cidForDocOrNull(adElement.getAmpDoc()).then(cidService => {
+  return getClientScopedAdCid(adElement.getAmpDoc(), adElement.win,
+                              config.clientIdScope, config.clientIdCookieName);
+}
+
+/**
+ * @param {!./service/ampdoc-impl.AmpDoc} doc
+ * @param {!Window} win
+ * @param {!string} clientIdScope
+ * @param {string=} opt_clientIdCookieName
+ * @return {!Promise<string|undefined>} A promise for a CID or undefined if
```

Comment:
Are you referring to that this and getAdCid have the same description of the return value? I don't think that's inaccurate